themes/custom/droopler_theme/scss/components/_d_p_text_blocks.scss

```diff
@@ -141,7 +142,11 @@ $d-p-single-text-box-grid-large-padding: $grid-gutter-width !default;
       justify-content: center;
 
       .content-wrapper {
+        text-align: center;
```

Comment:
This may cause an issue with CKEditor. Please check if "align" setting of the long text works properly for left/center/right. Probably right now "left" is selected by default, and it means "center" in reality. Some paragraph has a fix for that, please see "d_p_banner" for an example

Reply:
After testing I've removed the text-align property, looks like it can be handled by CKEditor, text stays in tiles no matter if we set it to 'left', 'right', or 'center'.